src/controller/data_model/controller-clusters.matter

```diff
@@ -4372,6 +4372,95 @@ provisional cluster DemandResponseLoadControl = 150 {
   command ClearLoadControlEventsRequest(): DefaultSuccess = 4;
 }
 
+/** This cluster provides an interface for passing messages to be presented by a device. */
+provisional cluster Messages = 151 {
+  revision 1;
```

Comment:
Messages is at revision 3 in the spec.

This results in several diffs I believe.

Reply:
Updated.